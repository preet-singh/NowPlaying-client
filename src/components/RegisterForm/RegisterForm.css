.Login {
    display: flex;
    flex-direction: column;
    margin-left: auto;
    margin-right: auto;
    border: 1px solid white;
    justify-content: center;
    color: white;
    background: rgba(0,0,0,0.7);
    border: none;
  }
  
.Login legend {
    font-size: 32px;
    color: #00d838;
    background: black;
    font-weight: bold;
}

.Login .black-button {
    width: 60%;
    margin: 0 auto;
}

.Login label {
    margin-top: 20px;
}

#username-input, #password-input, #email-input {
    border: none;
    background: none;
    border-bottom: 1px solid white;
    border-radius: 3px;
    text-align: left;
    padding-left: 10px;
    width: 60%;
    margin: 0 auto;
    margin-top: 5px;
    color: white;
}

#password-input {
margin-bottom: 10px;
}

.Login .login-link:hover {
color: #00d838;
}