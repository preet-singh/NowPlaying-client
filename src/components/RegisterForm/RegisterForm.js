//Dependencies
import React from 'react';
import { Link } from 'react-router-dom';

function RegisterForm() {
  return(
    <form className="Login" name="login">
      <legend>Register</legend>
      <label for="email-input" id="email-label" name="email-label">e-mail:</label>
      <input type="text" id="username-input" name="username-input" />
      <label for="username-input" id="username-label" name="username-label">username:</label>
      <input type="text" id="username-input" name="username-input" />
      <label for="password-input" id="password-label" name="password-label">password:</label>
      <input type="text" id="password-input" name="password-input" />
      <button type="submit" value="submit">Submit</button>
    </form>
  );
}

export default RegisterForm;