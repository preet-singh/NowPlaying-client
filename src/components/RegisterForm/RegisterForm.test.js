import React from 'react';
import RegisterForm from './RegisterForm.js';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`RegisterForm component`, () => {

  it('renders the RegisterForm', () => {
    const wrapper = shallow(<RegisterForm />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})