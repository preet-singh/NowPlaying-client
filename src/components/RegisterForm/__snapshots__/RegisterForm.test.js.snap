// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterForm component renders the RegisterForm 1`] = `
<form
  className="Login"
  name="register"
  onSubmit={[Function]}
>
  <legend>
    Register
  </legend>
  <label
    htmlFor="email-input"
  >
    E-Mail:
  </label>
  <input
    id="email-input"
    name="email"
    onChange={[Function]}
    placeholder="ex. john123@super.com"
    required={true}
    type="email"
  />
  <label
    htmlFor="username-input"
  >
    Username:
  </label>
  <input
    id="username-input"
    name="username"
    onChange={[Function]}
    placeholder="ex. John123"
    required={true}
    type="text"
  />
  <label
    htmlFor="password-input"
  >
    Password:
  </label>
  <input
    autoComplete="off"
    id="password-input"
    name="password"
    onChange={[Function]}
    placeholder="******"
    required={true}
    type="password"
  />
  <button
    className="black-button"
    id="reg_button"
    type="submit"
    value="submit"
  >
    Submit
  </button>
  <Link
    className="login-link"
    to="/login"
  >
    Already have an account?
  </Link>
</form>
`;
